src/types/indexer.ts

```diff
@@ -58,7 +58,8 @@ export type GetDelegatedStakingActivitiesResponse = GetDelegatedStakingActivitie
 export type GetCollectionDataResponse = GetCollectionDataQuery["current_collections_v2"][0];
 export type GetTokenDataResponse = GetTokenDataQuery["current_token_datas_v2"][0];
 export type GetProcessorStatusResponse = GetProcessorStatusQuery["processor_status"];
-export type GetFungibleAssetMetadataResponse = GetFungibleAssetMetadataQuery["fungible_asset_metadata"];
+export type GetFungibleAssetMetadatasResponse = GetFungibleAssetMetadataQuery["fungible_asset_metadata"];
+export type GetFungibleAssetMetadataResponse = GetFungibleAssetMetadataQuery["fungible_asset_metadata"][0];
```

Comment:
This can be... confusing at best.

Best to have it be `GetFungibleAssetMetadataResponse` and then `Array<GetFungibleAssetMetadataResponse>`

A single character inside a 32 character string is easy to miss

Reply:
So I think the problem here is that `GetFungibleAssetMetadataQuery["fungible_asset_metadata"]` is an array already - https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/types/generated/operations.ts#L470-L485